frontend/src/components/toast/toast-button.js

```diff
@@ -57,6 +57,10 @@ class ToastButton extends React.Component {
       justify-content: center;
     }
 
+    .button:focus{
```

Comment:
this is a small detail but try to be consistent with how the css is layed out (like put a space before the { and a new line before the next css class)